code/modules/reagents/reagent_dispenser.dm

```diff
@@ -1,331 +1,331 @@
-// Assuming this is http://en.wikipedia.org/wiki/Butane
-// (Autoignition temp 288�C, or 561.15�K)
-// Used in fueltanks exploding.
-#define AUTOIGNITION_WELDERFUEL 561.15
-
-/obj/structure/reagent_dispensers
-	name = "Dispenser"
-	desc = "..."
-	icon = 'icons/obj/objects.dmi'
-	icon_state = "watertank"
-	density = 1
-	anchored = 0
-	flags = FPRINT
-	pressure_resistance = 2*ONE_ATMOSPHERE
-
-	var/amount_per_transfer_from_this = 10
-	var/possible_transfer_amounts = list(10,25,50,100)
-
-/obj/structure/reagent_dispensers/attackby(obj/item/weapon/W as obj, mob/user as mob)
-	return
-
-/obj/structure/reagent_dispensers/examine(mob/user)
-	..()
+// Assuming this is http://en.wikipedia.org/wiki/Butane
```

Comment:
Git went absolutely nuclear trying to deff this file, what the fuck
http://i.imgur.com/S9seAaK.png